src/java/org/apache/cassandra/metrics/TableMetrics.java

```diff
@@ -934,7 +872,7 @@ public Long getValue()
 
         anticompactionTime = createTableTimer("AnticompactionTime", cfs.keyspace.metric.anticompactionTime);
         validationTime = createTableTimer("ValidationTime", cfs.keyspace.metric.validationTime);
-        syncTime = createTableTimer("SyncTime", cfs.keyspace.metric.repairSyncTime);
+        repairSyncTime = createTableTimer("RepairSyncTime", "RepairSyncTime", "SyncTime", cfs.keyspace.metric.repairSyncTime);
```

Comment:
`SyncTime` is new to 4.0 so don't need to worry about it https://issues.apache.org/jira/browse/CASSANDRA-13531

Also, SyncTime isn't present when I launch.


Also, 3 strings is confusing, feels like we should just do `createTableTimer("RepairSyncTime", cfs.keyspace.metric.repairSyncTime);`

Reply:
Ah, if this is new, I'll just simplify.